src/guides/v2.3/install-gde/install/legacy-file-system-perms.md

```diff
@@ -1,8 +1,8 @@
 ---
 group: installation-guide
 subgroup: 99_app
-title: Appendix&mdash;Magento file system ownership and appendix (legacy)
-menu_title: Appendix&mdash;Magento file system ownership and appendix (legacy)
+title: Appendix—Magento file system ownership and appendix (legacy)
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**